electrum_nmc/electrum/gui/qt/configure_dns_dialog.py

```diff
@@ -661,3 +676,113 @@ def update_byte_usage(self):
         label.setTextFormat(Qt.RichText)
         label.setText(usage_text)
 
+    def validate_address_input(self):
+        address_type = self.ui.comboHostType.currentText()
+        address = self.ui.editAHostname.text()
+
+        try:
+            validate_A_record(address, address_type)
+            self.ui.ErrorLabel.clear()
+        except Exception as e:
+            self.ui.ErrorLabel.setText(f"Warning: {e}")
+
+    def validate_cname_input(self):
```

Comment:
Lowercase `cname` is inconsistent with similar functions that use uppercase record types e.g. `NS`.